### ISSUE TITLE
Add MUC MAM support as a criteria for providers list

### ISSUE THREAD

**Issue by rdravi110** [xsf/xmpp-providers-website#66]
Hi, 

I think it is important to have group history in public channels, which is known as MUC MAM in XMPP jargon. In my experience, the listed sure.im provider does not have this feature.

**Comment by pravi** [comment]
Even though https://compliance.conversations.im/server/sure.im/ shows it support muc mam, we noticed rooms created on the server did not have this working.

**Comment by melvo** [comment]
Thanks for your ideas and testing! I created https://invent.kde.org/melvo/xmpp-providers/-/issues/75 for that since this repository is only for XMPP Providers' website.

**Comment by woj-tek** [comment]
Somewhat OT, but I just ran into the issue: @pravi - could you shed more light on it? For all we know Tigase MUC DOES support MAM history for MUC (and we do use it extensively…)?

**Comment by pravi** [comment]
@woj-tek we are not able to retrieve messages sent when we are offline from the server. Just confirmed this again. We created a test group, then I disabled my account, Ravi sent an image, I enabled and checked, no new messages.

![Screenshot_20231004-230522](https://github.com/xsf/xmpp-providers-website/assets/708222/942e450b-0342-4dba-816a-8849560c49da)

What I see ^

![20231004_231204031_bf17](https://github.com/xsf/xmpp-providers-website/assets/708222/50bd2afa-96b0-4d6b-bf80-73a0ab11fedf)

The actual muc state ^

This was on sure.im, this was private group, earlier we noticed missing messages in public muc too. I was using Monocles Chat which is a fork of conversations.
